api/routes.js

```diff
@@ -14,15 +14,14 @@ const upload = multer({
 
 router.get('/chatlist', controllers.getChatlist);
 router.get('/conversation', controllers.getConversation);
-// router.get('/example', auth, controllers.getExample);
 
-router.post('/addPhoto', upload.single('photo'), controllers.postAddPhoto);
-// router.post('/addMessage', auth, controllers.setExample);
-// router.post('/newChat', auth, controllers.setExample);
+router.post('/add-photo', upload.single('photo'), controllers.postNewPhoto);
```

Comment:
upload is the instantiation of multer with my provided options. this set up is checking the body for a key of photo with a value of an image file - then multer does its magic and grabs all of the data as it is sent to create the image file on the server. this can be accessed with req.file on the server side - we are using this to send the file to s3 for storage.